### PR TITLE
Fix brackets regex in column.py

### PR DESCRIPTION
**Pull request by guykoh**

### CODE REVIEW

File: dbt/adapters/clickhouse/column.py [hunk 1]
```diff
@@ -17,28 +17,26 @@ class ClickhouseColumn(Column):
         'INTEGER': 'Int64',
     }
     is_nullable: bool = False
-    _brackets_regex = re.compile(r'(Nullable|LowCardinality)\((.*?\))\)')
+    _brackets_regex = re.compile(r'^(Nullable|LowCardinality)\((.*?)\)$')
```
Comment (genzgd) [inline dbt/adapters/clickhouse/column.py:20]: Do we handle LowCardinality(Nullable(String)) types with this?

**Review by genzgd — APPROVED** [review]
LGTM, not sure the edge case of LowCardinality(Nullable) is actually important

**Comment by yoavo-datricks** [comment]
Hi @guykoh. I have a feeling this wouldn't work with map columns.

**Comment by guykoh** [comment]
> 

@yoavo-datricks, do you mean - https://clickhouse.com/docs/en/sql-reference/data-types/map? Why do you think it won't work?

**Comment by yoavo-datricks** [comment]
Hi.
Yeah, I thought that the fact it has two types will not work for the regex but it looks good.
Thanks for merging this!

**Pull request merged**